src/views/Login.js

```diff
@@ -0,0 +1,148 @@
+/*!
+
+=========================================================
+* Argon Dashboard React - v1.2.0
+=========================================================
+
+* Product Page: https://www.creative-tim.com/product/argon-dashboard-react
+* Copyright 2021 Creative Tim (https://www.creative-tim.com)
+* Licensed under MIT (https://github.com/creativetimofficial/argon-dashboard-react/blob/master/LICENSE.md)
+
+* Coded by Creative Tim
+
+=========================================================
+
+* The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+*/
+import React from "react";
+
+// reactstrap components
+import {
+  Button,
+  Card,
+  CardHeader,
+  CardBody,
+  FormGroup,
+  Form,
+  Input,
+  InputGroupAddon,
+  InputGroupText,
+  InputGroup,
+  Row,
+  Col,
+} from "reactstrap";
+import axios from 'axios';
+import {Redirect} from "react-router-dom";
+import {BASE_URL} from '../base_url.js';
+
+class Login extends React.Component {
+
+
+    constructor(props) {
+        super(props);
+        this.state = {email: '', password: '', loggedin: false};
+    }
+
+    handleChange = (event) => {
+        this.setState({[event.target.name]: event.target.value, [event.target.password]: event.target.value});
+    };
+
+    handleSubmit = async (event) => {
+        event.preventDefault();
+        await axios.post(
```

Comment:
should try and integrate `fetch` instead

Reply:
Okay